reports/RJ722.md

```diff
@@ -146,26 +146,26 @@ activity:
 
 <br>
 
-1.) In addition to the first line of unused code, Vulture now reports the last 
+1.) In addition to the first line of unused code, Vulture now reports the last
 line of unused code. The results can now also be sorted according to their size.
 
 <br>
 
-2.) Other than the conventional dead code (defined but not used), vulture 
+2.) Other than the conventional dead code (defined but not used), vulture
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.